### ISSUE TITLE
iterm 3 opens in new tab + the default

### ISSUE THREAD

**Issue by ctf0** [wbond/sublime_terminal#131]
currently when u fire the plugin, it will open iterm with 2 tabs like so

![test](https://cloud.githubusercontent.com/assets/7388088/17993529/be87f6f6-6b50-11e6-854f-885e565859a5.jpg)

also the script needs to activate the terminal after its opened.

**Comment by twolfson** [comment]
Which iTerm script are you using?

**Comment by twolfson** [comment]
I believe this is a duplicate of #132 which has been patched via #133. Closing issue as a result

**Comment by ctf0** [comment]
still exist, nothing changed :( & am using the latest included script for `iTerm2-v3`

**Comment by garyking** [comment]
Is it possible that you have iTerm running but it just doesn't have any windows open? So it has the black dot underneath it in the Dock?

**Comment by ctf0** [comment]
nop.

**Comment by ctf0** [comment]
update
the bringing iterm window forward now is fixed, now the double tab issue
btw if there a specific option i should check/uncheck in iterm please tell me.

**Comment by twolfson** [comment]
Can you paste your configuration and your version of Terminal (found via Package Control: List Packages in the Command Palette)?

**Comment by ctf0** [comment]
sure
```json
{
    "terminal": "iTerm2-v3.sh"
}
```
![test](https://cloud.githubusercontent.com/assets/7388088/18195803/fb808468-70ed-11e6-81ac-81a9e2533423.jpg)

**Comment by twolfson** [comment]
Okay, everything looks good. You have explained the unexpected behavior but can you explain what you expect?

**Comment by ctf0** [comment]
open in one tab without the extra default :smiley:

**Comment by twolfson** [comment]
When you run the plugin is iTerm already open?

**Comment by ctf0** [comment]
sorry for the delayed answer, no

**Comment by twolfson** [comment]
Alright, going to re-open this issue and mark as "help wanted" as I don't have an OS X machine to test/reproduce the issue on =/ But it sounds like the steps to reproduce are:

- Close/quit all iTerm2 instances
- Configure Terminal plugin with `{"terminal": "iTerm2-v3.sh"}`
- Open a file
- Run "Terminal: Open" command or shortcut

Expected behavior:

- iTerm2 launches with a single tab in file's directory

Actual behavior:

- iTerm2 launches with a tab in its default directory and another tab in file's directory

**Comment by ctf0** [comment]
thanx

**Comment by ctf0** [comment]
@twolfson currently if another windows was already opened, the plugin work as expected `open a new window for the ST path` ,
maybe u can add like a hook which check if there was no opened window so the plugin will close any tabs other than the one with the path from ST

**Comment by twolfson** [comment]
As I said I don't have an OS X machine so I can't do anything beyond marking this as "help wanted". Feel free to take a shot at patching the issue though :+1:

**Comment by crapthings** [comment]
i can confirm that if i press Ctrl+Q with iterm, there will be two tabs, one is default, and the second is opened with this terminal plugin.

**Comment by crapthings** [comment]
so if you want this work. you might have to change iterm preferences

startup Don't open any windows, but i think this is not convenient.

@ctf0

**Comment by ctf0** [comment]
@crapthings am tyring to make iterm work as the default terminal work
- have a default path to start with when u open terminal 
- if we specified a path to start with ex.`through this package` the default is overridden and we get only once windows

so am not sure what exactly is the option i should change in iterm to achive that but i dont think this is related to iterm because of https://github.com/wbond/sublime_terminal/issues/131#issuecomment-249361290

**Comment by crapthings** [comment]
is there something like "open with external" with iterm ?

**Comment by ctf0** [comment]
under which settings ?

**Comment by plashenkov** [comment]
It seems that this line: https://github.com/wbond/sublime_terminal/blob/master/iTerm2-v3.sh#L36 is unnecessary. When iTerm starts, it already opens a tab. If you comment the line, everything becomes as it should.

**Comment by ctf0** [comment]
@plashenkov fuck yeah 💃

**Comment by crapthings** [comment]
so we will get this fix soon ?

**Comment by twolfson** [comment]
I don't have a OS X device to test on but will gladly review any PR that someone else has tested

**Comment by ctf0** [comment]
@twolfson comment the line and test if u have any issues on ur machine, if all good i will submit the PR

**Comment by twolfson** [comment]
I don't have an OS X device so commenting the line won't really affect me as it's an OS X specific script =/

**Comment by ctf0** [comment]
sweet, then i'll submit the PR

**Comment by plashenkov** [comment]
It looks like some mistake is above.

Look: here we get version: https://github.com/wbond/sublime_terminal/blob/master/iTerm2-v3.sh#L7

Here we check version is less or larger than `10.7`: https://github.com/wbond/sublime_terminal/blob/master/iTerm2-v3.sh#L21

And despite the fact my version is 10.12 it acts like if it less than 10.7. This is why 
algorithm later goes in the wrong direction. Or maybe I missed something?

**Comment by ctf0** [comment]
i will reopen it till further notice

**Comment by twolfson** [comment]
It looks like that's a hold-over from `iTerm.sh` that @pzgz never removed =/ @pzgz Was leaving that `VERSION` check in intentional?

https://github.com/wbond/sublime_terminal/blob/1.14.0/iTerm.sh#L21-L30

**Comment by plashenkov** [comment]
Well, I do not know. If we remove this part: https://github.com/wbond/sublime_terminal/blob/master/iTerm2-v3.sh#L22-L25 and leave this part: https://github.com/wbond/sublime_terminal/blob/master/iTerm2-v3.sh#L29 then we still get double tab, and logic below [this line](https://github.com/wbond/sublime_terminal/blob/master/iTerm2-v3.sh#L46) needs corrections.

**Comment by plashenkov** [comment]
It's interesting, does [this comparison](https://github.com/wbond/sublime_terminal/blob/master/iTerm2-v3.sh#L21) actually work as the author of these lines was expected.

```
VERSION=10.12

if (( $(expr $VERSION '<' 10.7) )); then
    echo "$VERSION < 10.7"
else
    echo "$VERSION > 10.7"
fi
```

10.12 < 10.7
10.8 > 10.7
2 > 10.7
etc.

**Comment by twolfson** [comment]
This has been patched by @ctf0 in #131 and released in `1.15.0`. If we still have issues, then please let us know
